pooranjoyb/popShop
# added Return/Exchange feature

Fixes #299 

## Type of change

Please give a X on it which is applicable

- [X] Others(mentioned in the issue number)

# How Has This Been Tested?

1. Go to 'Orders'
2. Click on the `View` Button.
3. Choose the option from the dropdown.
4. Choose return/exchange.
5. Add details.
6. Submit.
7. Checks are added. Check by submitting without any details.

# Screenshots and Videos:
![image](https://github.com/pooranjoyb/popShop/assets/127378920/089aac73-e141-4dfb-b33a-4a650d383fa1)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/903ff57a-3ba9-4adc-9dbc-6871eced7e9d)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/6abdc96d-7100-4c32-b314-7d546879a95d)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/45677dc3-fd06-4bfe-aa10-506a82051cb1)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/579e30a7-6bd6-4822-b7cc-cfeedd39db1d)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/5da61aca-5005-4ef8-9487-ad5215366cca)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/bdef6ec8-ad10-47b7-a9d3-642c2faaaf37)
CONSOLE:
![image](https://github.com/pooranjoyb/popShop/assets/127378920/de154571-8534-4490-b831-35b4fe3e44fe)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/2da923f3-ade9-4a7b-8766-552eb2ad480d)


# Checklist:
Give a X on, which is applicable
